examples/conformer_energies/conformer_energies.py

```diff
@@ -0,0 +1,93 @@
+        integrator = openmm.VerletIntegrator(1*unit.femtoseconds)
+        platform = openmm.Platform.getPlatformByName('Reference')
+        omm_top = off_top.to_openmm()
+        simulation = openmm.app.Simulation(omm_top, system, integrator, platform)
+
+        # Print text header
+        print('Conformer         Initial PE         Minimized PE       RMS between initial and minimized conformer')
+        output = [['Conformer','Initial PE (kcal/mol)','Minimized PE (kcal/mol)','RMS between initial and minimized conformer (Angstrom)']]
+        for conformer_index, conformer in enumerate(molecule.conformers):
+            simulation.context.setPositions(conformer)
+            orig_potential = simulation.context.getState(getEnergy=True).getPotentialEnergy()
+            simulation.minimizeEnergy()
+            min_state = simulation.context.getState(getEnergy=True, getPositions=True)
+            min_potential = min_state.getPotentialEnergy()
+
+            # Calculate the RMSD between the initial and minimized conformer
+            min_coords = min_state.getPositions()
+            min_coords = np.array([ [atom.x, atom.y, atom.z] for atom in min_coords]) * unit.nanometer
+            mol_copy._conformers = None
+            mol_copy.add_conformer(conformer)
+            mol_copy.add_conformer(min_coords)
+            rdmol = mol_copy.to_rdkit()
+            rmslist = []
+            rdMolAlign.AlignMolConformers(rdmol, RMSlist=rmslist)
+            minimization_rms = rmslist[0]
+
+            # Save the minimized conformer to file
+            mol_copy._conformers = None
+            mol_copy.add_conformer(min_coords)
+            mol_copy.to_file(f'{molecule.name}_conf{conformer_index+1}_minimized.sdf', file_format='sdf')
+            print('%5d / %5d : %8.3f kcal/mol %8.3f kcal/mol  %8.3f Angstroms' % (conformer_index+1, molecule.n_conformers, orig_potential/unit.kilocalories_per_mole, min_potential/unit.kilocalories_per_mole, minimization_rms))
+            output.append([str(conformer_index+1), 
+                           f'{orig_potential/unit.kilocalories_per_mole:.3f}', 
+                           f'{min_potential/unit.kilocalories_per_mole:.3f}', 
+                           f'{minimization_rms:.3f}'])
+            # Write the results out to CSV
+        with open(f'{molecule.name}.csv', 'w') as of:
+            for line in output:
+                of.write(','.join(line)+'\n')
+                # Clean up OpenMM Simulation
+        del simulation, integrator
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(description="Perform energy minimization on a molecule, potentially with many conformers. For each conformer, this script will provide the initial energy, minimized energy, and RMSD between the initial and minimized structure (both as STDOUT and a csv file). The minimized conformers will be written out to SDF.")
+    parser.add_argument('-f', '--filename', help='Name of an input file containing conformers')
```

Comment:
I'd allow arguments like `--molecules molecules.sdf` for input molecules, `--output output.{sdf|csv}`, and optionally `--forcefield openff-1.0.0`.